Kitodo-API/src/main/java/org/kitodo/api/dataformat/mets/KitodoUUID.java

```diff
@@ -0,0 +1,66 @@
+package org.kitodo.api.dataformat.mets;
```

Comment:
License header is missing.

Reply:
Thanks for the hint. Actually, this should be intercepted by Checkstyle, which is why I am surprised that it passed all checks.

Reply:
Maybe because there was less errors then allowed errors and so this error was not visible. I reduced the amount of errors and allowed errors in last pull request, so rebasing this branch should be a good idea.